File: Java/Arrays_and_Strings/Reverse.java
class Innoskrit {
	static void reverse(int[] arr)
	{
        int start = 0, end = arr.length - 1;
        
        while(start < end) {
            swap(arr, start, end);
            start++;
            end--;
        }
        
        for(int element : arr) {
            System.out.print(element + " ");
        }
	}

	static void swap(int[] arr, int i, int j) {
		int temp = arr[i];
		arr[i] = arr[j];
		arr[j] = temp;
	}
	
	public static void main(String args[])
	{
		int[] arr = {1, 2, 3, 4, 5, 6, 7};
        reverse(arr);
	}
}

File: Python/Arrays_and_Strings/Reverse.py
def reverse(arr):

	start, end = 0, len(arr) - 1
	
	while(start < end):
		temp = arr[start]
		arr[start] = arr[end]
		arr[end] = temp
		
		start += 1
		end -= 1
	
	for i in range(len(arr)):
		print(arr[i], end = " ")

if __name__ == '__main__':
	arr = [1, 2, 3, 4, 5, 6, 7]
	reverse(arr)